src/cryptsetup/cryptsetup-generator.c

```diff
@@ -46,30 +46,30 @@ STATIC_DESTRUCTOR_REGISTER(arg_disks, hashmap_freep);
 STATIC_DESTRUCTOR_REGISTER(arg_default_options, freep);
 STATIC_DESTRUCTOR_REGISTER(arg_default_keyfile, freep);
 
-static int split_keyspec(const char *keyspec, char **keyfile, char **keydev) {
+static int split_keyspec(const char *keyspec, char **ret_keyfile, char **ret_keydev) {
```

Comment:
With this renaming, it'd be nice to also rename the local variables to `keyfile` and `keydev`. But this can be done later...